packages/rollup-contracts/contracts/ExecutionManager.sol

```diff
@@ -1019,9 +995,13 @@ contract ExecutionManager is FullStateManager {
      * @return The L1MessageSender in our current execution context.
      */
     function getL1MessageSender() public returns(address) {
-        require(executionContext.ovmActiveContract == l1MsgSenderAddress, "Only the L1MessageSender precompile is allowed to call getL1MessageSender(...)!");
+        /* require(executionContext.ovmActiveContract == l1MsgSenderAddress, "Only the L1MessageSender precompile is allowed to call getL1MessageSender(...)!"); */
```

Comment:
Is there a reason this is commented out?  I can't see any problem with dropping this restriction, but we should just remove it if that's how we wanna go.

Reply:
Wow good catch. Looks like Mason commented that out a long time ago when debugging but forgot to uncomment. Fixing now